ui/src/components/helpers/lists.js

```diff
@@ -0,0 +1,333 @@
+const accessTypeList = ["Unsigned", "Signed", "Alumni", "Admin", "Stewardship"];
```

Comment:
not used anywhere?

Reply:
for future, the lists.js is for everyone to use and not just forms